app/controllers/admin/csv_controller.rb

```diff
@@ -28,6 +28,17 @@ def regular_schedules; end
 
     def services; end
 
+    def upload_services
+      upload_services_params
+      importer = ServiceUploader.new(params[:services].tempfile)
+      importer.process
+      flash[:notice] = t('admin.services.upload.success')
+      redirect_to admin_csv_downloads_path
```

Comment:
Can combine the `notice` into the redirect here as is done in the `ContactsController` above